### PR TITLE
Removed Archived YouTube Videos From Pages

### PR DESCRIPTION
**Pull request by RileySeaburg**
## Summary

Remove archived YouTube videos from content. Closes [#7528].

| pages with youtube video removed |
|------------------------------------|
| content/news/2013/02/2013-02-11-youll-have-them-at-swipe-making-an-awesome-mobile-user-experience-webinar.md |
| content/news/2013/02/2013-02-14-qr-codes.md |
| content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md |
| content/news/2018/04/2018-04-02-marketing-government-through-agency-partnerships.md |
| content/news/2016/07/2016-07-14-analytics-trainings-2016-so-far.md |
| content/news/2016/05/2016-05-12-using-pirate-metrics-to-analyze-your-mobile-applications-audience.md |
| content/news/2017/09/2017-09-20-a-conversation-with-itif-about-state-federal-government-websites.md |
| content/resources/digitalgov-user-experience-program-speaking-opportunities.md |
| content/resources/digitalgov-user-experience-program-speaking-opportunities.md |
| content/resources/inventory-data-gov-guide.md |
### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/)

https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2013/02/2013-02-11-youll-have-them-at-swipe-making-an-awesome-mobile-user-experience-webinar
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2013/02/2013-02-14-qr-codes
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages-creators
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2018/04/2018-04-02-marketing-government-through-agency-partnerships
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2016/07/2016-07-14-analytics-trainings-2016-so-far
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2016/05/2016-05-12-using-pirate-metrics-to-analyze-your-mobile-applications-audience
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/news/2017/09/2017-09-20-a-conversation-with-itif-about-state-federal-government-websites
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/resources/digitalgov-user-experience-program-speaking-opportunities
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/resources/digitalgov-user-experience-program-speaking-opportunities
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-deleted-videos-from-pages/resources/inventory-data-gov-guide

### How To Test

1. First Step
2. Second Step
3. Third Step

---

### Dev Checklist

- [ ] PR has correct labels
- [ ] A11y testing (voice over testing, meets WCAG, run axe tools)
- [ ] Code is formatted properly

### CODE REVIEW

File: content/news/2013/02/2013-02-14-qr-codes.md [hunk 1]
```diff
@@ -39,8 +39,7 @@ Good Practice Tip: Keep in mind that QR codes that lead to mobile-friendly conte
 
 ## <a name="x-Training"></a>Training {#toc2}
 
-<!-- TODO: this video does not exist -->
-{{< youtube id="22i4hQCNsjM" >}}
+<!-- TODO: this video does not exist -->
```
Comment (nick-mon1) [inline content/news/2013/02/2013-02-14-qr-codes.md:42]: Let's remove this comment.

File: content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md [hunk 2]
```diff
@@ -10,9 +10,13 @@ topics:
 
 ---
 
+## Archive Notice
+
+This page is no longer being updated. It may contain outdated information and links may no longer function as originally intended.
+
```
Comment (nick-mon1) [inline content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md:16]: We can remove this and use the archive warning banner instead.

File: content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md [hunk 3]
```diff
@@ -25,8 +29,6 @@ Fuller states that when creating quizzes the “Bottom line is, go make somethin
 
 Not all quiz questions come answered straight out of the box. When using social media quizzes, organization must still consider issues such as 508 compliance. However, results can be achieved quickly. As Owen cautions, “A really good quiz takes maybe 1 to 2 hours to create, not because of the interface but because of the thinking behind it.”
 
-To learn more about quizzes, you can view the entire webinar.
-
-{{< youtube id="0q4ueTABYnQ" >}}
+To learn more about quizzes, you can view the entire webinar.
```
Comment (nick-mon1) [inline content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md:32]: Let's remove this.

File: content/news/2016/07/2016-07-14-analytics-trainings-2016-so-far.md [hunk 4]
```diff
@@ -9,15 +9,19 @@ topics:
   - analytics
 ---
 
+## Archive Notice
+
+This page is no longer being updated. It may contain outdated information and links may no longer function as originally intended.
+
```
Comment (nick-mon1) [inline content/news/2016/07/2016-07-14-analytics-trainings-2016-so-far.md:15]: We can remove this and use the archive warning banner instead.

File: content/news/2017/09/2017-09-20-a-conversation-with-itif-about-state-federal-government-websites.md [hunk 5]
```diff
@@ -22,9 +22,11 @@ primary_image: "itif-presentation-title-slide"
 
 ---
 
-At the beginning of 2017, the ITIF (Information Technology and Innovation Foundation) released [a report](https://www.digitalgov.gov/2017/03/24/new-itif-report-inspires-a-closer-look-at-website-performance-and-security-here-is-where-to-begin/) that benchmarked 300 federal websites in four areas: page-load speed, mobile friendliness, security and accessibility. Some sites fared better than others, but the report highlighted that our federal sites have a ways to go (DigitalGov included) in these areas.
+## Archive Notice
 
-{{< youtube id="Mu-MXrkv0sw" >}} 
+This page is no longer being updated. It may contain outdated information and links may no longer function as originally intended.
+
```
Comment (nick-mon1) [inline content/news/2017/09/2017-09-20-a-conversation-with-itif-about-state-federal-government-websites.md:28]: We can remove this and use the archive warning banner instead.

File: content/resources/digitalgov-user-experience-program-speaking-opportunities.md [hunk 6]
```diff
@@ -9,6 +9,10 @@ authors:
   - andreanocesigritz
 ---
 
+## Archive Notice
+
+This page is no longer being updated. It may contain outdated information and links may no longer function as originally intended.
+
```
Comment (nick-mon1) [inline content/resources/digitalgov-user-experience-program-speaking-opportunities.md:15]: We can remove this and use the archive warning banner instead.

**Review by nick-mon1 — CHANGES_REQUESTED** [review]

**Comment by RileySeaburg** [comment]
@nick-mon1  I think I addressed you requests but Idk if I checked all the boxes to say I did.

**Comment by nick-mon1** [comment]
@RileySeaburg We should have a link to each preview page to make testing easier. I've tested locally and everything looks good.

File: package-lock.json
Comment (nick-mon1) [inline package-lock.json:1]: We should exclude these changes, we update packages for security/maintenance in separate PR's.

**Review by nick-mon1 — CHANGES_REQUESTED** [review]

**Review by nick-mon1 — APPROVED** [review]

**Comment by RileySeaburg** [comment]
Solves #7538

File: content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md
Comment (ToniBonittoGSA) [inline content/news/2016/03/2016-03-08-quizzes-the-secret-weapon-of-the-worlds-best-content-creators.md:1]: Hi @RileySeaburg 

Original:

> `[Quizzes: The Secret Weapon of the World’s Best Content Creators](https://www.youtube.com/watch?v=0q4ueTABYnQ),`

was changed to 

> `[This link no longer exists](https://www.youtube.com/watch?v=0q4ueTABYnQ),`

The copy "This link no longer exists" removes context, doesn't tell user **what** no longer exists. Also, we wouldn't link to something that no longer exists. Pls use:

> `Quizzes: The Secret Weapon of the World’s Best Content Creators,`

File: content/news/2016/05/2016-05-12-using-pirate-metrics-to-analyze-your-mobile-applications-audience.md
Comment (ToniBonittoGSA) [inline content/news/2016/05/2016-05-12-using-pirate-metrics-to-analyze-your-mobile-applications-audience.md:1]: @RileySeaburg 
The banner alerts users that content is older and links may no longer work. 
Remove the link, but keep the copy. Restore: 

> Watch the webinar to see how you can learn how to use pirate metrics in your agency!

File: content/news/2016/07/2016-07-14-analytics-trainings-2016-so-far.md
Comment (ToniBonittoGSA) [inline content/news/2016/07/2016-07-14-analytics-trainings-2016-so-far.md:1]: @RileySeaburg 

Like the others; pls restore the "## DAP 101 Training" section; just remove the links.

File: content/resources/digitalgov-user-experience-program-speaking-opportunities.md
Comment (ToniBonittoGSA) [inline content/resources/digitalgov-user-experience-program-speaking-opportunities.md:1]: @RileySeaburg 

Like the others; restore "## Our Recorded Webinars" section, just remove the vids

File: content/resources/inventory-data-gov-guide.md
Comment (ToniBonittoGSA) [inline content/resources/inventory-data-gov-guide.md:1]: @RileySeaburg 

For 
> Data.gov, working with DigitalGov University, presented a webinar for agencies interested in using inventory on December 16, 2014\. For more information, consult the recording of the [This link is no longer maintained](http://youtu.be/ciLYjDsF4lo).

Use

> Data.gov, working with DigitalGov University, presented a webinar for agencies interested in using inventory on December 16, 2014. For more information, consult the recording of the webinar.

(also removes odd backslash at end of the first sentence)

Thx!

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**